pkg/virt-handler/realtime.go

```diff
@@ -0,0 +1,196 @@
+//go:build linux && amd64
+
+package virthandler
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+
+	// #include <linux/sched.h>
+	// #include <linux/sched/types.h>
+	// typedef struct sched_param sched_param;
+	"C"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+type schedParam C.sched_param
+type policy uint32
+type maskType bool
+
+const (
+	SCHED_FIFO policy   = C.SCHED_FIFO
```

Comment:
nit: I'd change this to `schedFifo` to better align with Go's convention and to not expose this variable

Reply:
Makes sense. I like the Go coding conventions :smile: , but in this case I'll take `schedFIFO` since it's an acronym.

Reply:
makes sense :)